testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDatabricksWithGlueMetastoreCleanUp.java

```diff
@@ -100,19 +105,20 @@ private void cleanSchema(String schema, long startTime, AWSGlueAsync glueClient)
         }
     }
 
-    private Set<String> findAllTestTablesInSchema(String schema)
+    private Set<String> findAllTablesInSchema(String schema)
     {
         try {
             QueryResult allTables = onTrino().executeQuery(format("SHOW TABLES IN %s", schema));
-            log.info("Found %d potential tables to drop in schema %s", allTables.rows().size(), schema);
             return allTables.rows().stream()
                     .map(row -> (String) row.get(0))
-                    .filter(name -> name.toLowerCase(Locale.ROOT).startsWith("test"))
                     .collect(Collectors.toUnmodifiableSet());
         }
         catch (Exception e) {
-            log.warn(e, "Exception while fetching tables for schema %s", schema);
-            return ImmutableSet.of();
+            // The schema could be removed concurrently
+            if (getStackTraceAsString(e).contains("Schema '%s' does not exist".formatted(schema))) {
+                return ImmutableSet.of();
+            }
+            throw e;
```

Comment:
we used to swallow, not throw. is the change intentional?

Reply:
It was intentional, but reverted as a part of `information_schema.tables` change.